reframe/frontend/printer.py

```diff
@@ -48,15 +48,15 @@ def status(self, status, message='', just=None, level=logging.INFO):
 
         status_stripped = status.strip()
         if self.colorize:
-            if status_stripped == 'SKIP':
+            if status_stripped in ('DRY', 'SKIP'):
                 status = color.colorize(status, color.YELLOW)
-            elif status_stripped in ['FAIL', 'FAILED', 'ERROR']:
+            elif status_stripped in ('FAIL', 'FAILED', 'ERROR'):
                 status = color.colorize(status, color.RED)
             else:
                 status = color.colorize(status, color.GREEN)
 
         final_msg = f'[ {status} ] '
-        if status_stripped in ['OK', 'SKIP', 'FAIL']:
```

Comment:
is `'DRY'` needed here as well?

Reply:
No, the counter is increased to have an output like:
```bash
...
[ DRY      ] jupyterhub_submit_test /45d367fb @daint:jupyter_mc+builtin
[ DRY      ] NCO_MergeNC4CTest /6636720f @daint:gpu+builtin
[ DRY      ] NCO_MergeNC4CTest /6636720f @daint:mc+builtin
[       OK ] (  5/890) nvidia_smi_check %gpu_mode=ecc /78390e2f @daint:gpu+builtin
[       OK ] (  6/890) nvidia_smi_check %gpu_mode=compute /a936ce18 @daint:gpu+builtin
...
```
Only when the test is completed we want to increase the counter with the completed tests

Reply:
Nope, that's just the outcome of the test. We are not printing the `DRY` there :-)